src/components/ui/form-textarea/FormTextarea.vue

```diff
@@ -1,26 +1,22 @@
 <script setup lang="ts">
 import { ref } from 'vue'
 import FormWrapper from '../form-wrapper/FormWrapper.vue'
+import type { InputMode } from 'src/types'
```

Comment:
aqui devemos usar o caminho direto
```suggestion
import type { InputMode } from '../../../types'
```